src/System.Private.CoreLib/shared/System/IO/Stream.cs

```diff
@@ -632,10 +635,12 @@ private sealed class ReadWriteTask : Task<int>, ITaskCompletionAction
                 }
             }
 
-            private static void InvokeAsyncCallback(object completedTask)
+            private static void InvokeAsyncCallback(object? completedTask)
```

Comment:
Why is this nullable? I know is private, but it would also fail with `null` right?

Reply:
Same reason as Cancel earlier: it needs to be to match the delegate signature.